src/node/mqtt.rs

```diff
@@ -53,8 +53,8 @@ impl Topic {
           [
             Regex::new(r"messages/([A-Fa-f0-9]{64})/metadata").expect("regex failed"),
             Regex::new(r"outputs/([A-Fa-f0-9]{64})(\d{4})").expect("regex failed"),
-            // bech32 address
-            Regex::new("addresses/(iota|atoi|iot|toi)1[A-Za-z0-9]+/outputs").expect("regex failed"),
+            // BIP-173 compliant bech32 address
+            Regex::new("addresses/[\x22-\x7E]{1,30}1[A-Za-z0-9]+/outputs").expect("regex failed"),
```

Comment:
https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#:~:text=addresses%20using%20it.-,Bech32,range%20%5B33%2D126%5D.

Shouldn't it be `[\x21-\x7E]`?

Reply:
Yep, you're right! I changed it and tested it.